fe/fe-core/src/main/java/com/starrocks/journal/bdbje/BDBEnvironment.java

```diff
@@ -92,7 +92,7 @@ public class BDBEnvironment {
     // wait at most 10 seconds after environment initialized for state change
     private static final int INITAL_STATE_CHANGE_WAIT_SEC = 10;
 
-    public static final String STARROCKS_JOURNAL_GROUP = "PALO_JOURNAL_GROUP";
+    public static final String STARROCKS_JOURNAL_GROUP = "STARROCKS_JOURNAL_GROUP";
```

Comment:
@gengjun-git plz help to confirm whether will it cause any compatibility issue.

Reply:
Yes, this cannot be changed.

Reply:
done